python/aswfdocker/data/ci-image-dockerfile.jinja2

```diff
@@ -27,6 +27,11 @@ ARG {{ arg }}
 {% endfor -%}
 {% endfor %}
 
+{% if name == "common" %}
```

Comment:
I'm guessing the `if` will drop when you start building other ci images?

Reply:
I don't remember why I made the code conditional, or why it's where it is. I will def revisit, thanks for pointing it out. I think I took inspiration on:

```
{% if name != "common" %}
ARG ASWF_VFXPLATFORM_VERSION
{% endif %}
```

earlier in the file. I guess the question is whether it's important to minimize the number of ARG statements or not?

Reply:
Looking closer: only ci-common/Dockerfile needs those arguments to reference the correct NVIDIA base image, other images don't.